certbot/README.md

```diff
@@ -1,8 +1,32 @@
 # Less-manual manual cert rotation tools
 
-## Usage
+## all-in-one
 
-Both scripts expect you have set the environment variable CERTBOT_BUCKET_NAME to be the name
+*Note* This uses bash-isms not available on the default OS X bash (3.x). 
```

Comment:
Missing words:

```suggestion
*Note* This uses bash-isms, which is not available on the default OS X bash (3.x). 
```

Reply:
this suggestion doesn't work, but it does tell me that my original wording isn't great, so I'll tweak it

Reply:
No worries!